frontend/js/services/Store.service.js

```diff
@@ -17,6 +33,12 @@ const Store = (function() {
     }
 
     return {
+        setModalId : function(id) {
+            modalId = id;
+        },
+        getModalId : function() {
+            return modalId;
+        },
         getinstance: function() {
```

Comment:
getInstance will be the frst method here .. please add method in the bottom incrementally or group them with common functionality

Reply:
getModalId is removed now.. since it was not being called